libs/core---samd/platform.h

```diff
@@ -135,4 +135,73 @@ typedef int PinName;
 #define PB30 62
 #define PB31 63
 
+
+// 128 pin packages
+#ifdef SAMD51
```

Comment:
@mmoskal i assume we always want those so probably remove this #ifdef

Reply:
yes